package.json

```diff
@@ -14,14 +15,19 @@
     "babel-loader": "^7.1.5",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^1.0.0",
+    "html-webpack-plugin": "^3.2.0",
+    "optimize-js-plugin": "0.0.4",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
+    "react-hot-loader": "^4.3.3",
     "style-loader": "^0.21.0",
-    "webpack": "^4.15.1",
-    "webpack-cli": "^3.0.8"
+    "webpack": "^4.15.1"
+
   },
   "dependencies": {
     "babel-preset-react": "^6.24.1",
-    "uuid": "^3.3.2"
+    "uuid": "^3.3.2",
+    "webpack-cli": "^3.0.8",
+    "webpack-dev-server": "^3.1.4"
```

Comment:
w `dependencies` umieszczasz to, od czego zalezy produkcyjne dzialanie aplikacji, tutaj bedzie to tylko `"uuid"`, reszta do `devDependencies` poniewaz uzywasz ich tylko developersko do wyprodukowania paczki